### DISCUSSION TITLE
Can't find the modal component on chakra UI v3?

### DISCUSSION THREAD

**Discussion by techwithtwin** [chakra-ui/chakra-ui#9312] [General] [answered]
Was the modal component removed in v3, if so what is the alternative use?

**Comment by techwithtwin** [comment]
or the workaround is to use stuff like react modal?

**Comment by veloware** [comment, marked as answer]
Its been renamed to Dialog
https://www.chakra-ui.com/docs/components/dialog

All the other v2 -> v3 differences are documented in here too - https://www.chakra-ui.com/docs/get-started/migration
